driver-extras/pom.xml

```diff
@@ -142,18 +142,39 @@
             <plugin>
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>animal-sniffer-maven-plugin</artifactId>
-                <configuration>
-                    <!--
-                    There is no support for JDK 8 yet, see
-                    https://github.com/mojohaus/animal-sniffer/issues/1
-                    -->
-                    <skip>true</skip>
-                    <signature>
-                        <groupId>org.codehaus.mojo.signature</groupId>
-                        <artifactId>java18</artifactId>
-                        <version>1.0</version>
-                    </signature>
-                </configuration>
+                <executions>
```

Comment:
Took me a little bit to wrap my head around this, but this is very cool.  The annotation solution is pretty elegant.  It would be even nicer if `animal-sniffer` supported package level annotations (i.e. if a package is annotated, it skips all of its classes), but it doesn't look like that currently works (but maybe a nice change request for animal sniffer).